src/main/java/net/openhft/chronicle/core/Memory.java

```diff
@@ -378,7 +378,7 @@ default long heapUsed() {
          * @param length  the number of bytes to copy
          * @deprecated This method is deprecated and will be removed in a future version (x.24).
          */
-    @Deprecated(/* to be removed in x.24 */)
+    @Deprecated(/* to be removed in x.26 */)
```

Comment:
I'm not sure this will help. You cannot remove this method without removing the rest of copyMemory() ones, since a different overload will be quietly used which has different behavior (offset for object header v.s. not offset).
See https://github.com/OpenHFT/Chronicle-Core/pull/603